v2/AdminCore.WebApp/Controllers/HolidayController.cs

```diff
@@ -73,8 +73,7 @@ public IActionResult GetHolidayByEmployeeId()
         return Ok(_mapper.Map<IList<HolidayViewModel>>(holiday));
       }
 
-      return StatusCode((int)HttpStatusCode.NoContent, $"No holiday found for employee ID: { employeeId.ToString() }");
-
+      return StatusCode((int)HttpStatusCode.NoContent, $"No holiday found for employee ID: { _authenticatedUser.RetrieveUserId().ToString() }");
```

Comment:
There's no point of calling the Retrieve service twice. Can you initialise with a variable at the top